dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.unboundid.ad.ssl/server.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http:\\www.eclipse.org\legal\epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="com.ibm.ws.security.registry.ldap.fat.unboundid.ssl">
+
+	<include location="../fatTestPorts.xml"/>
+
+    <!-- Enable features -->
+    <featureManager>
+        <feature>appSecurity-2.0</feature>
+        <feature>servlet-3.1</feature>
+        <feature>securitylibertyinternals-1.0</feature>
+		<feature>ldapRegistry-3.0</feature>
+    </featureManager>
+    
+    <ldapRegistry id="ldap" realm="SampleLdapADRealm" host="${ldap.server.2.name}" port="${ldap.server.2.ssl.port}" ignoreCase="true"  baseDN="DC=SECFVT2,DC=AUSTIN,DC=IBM,DC=COM"
+		bindDN="cn=testuser,cn=users,dc=secfvt2,dc=austin,dc=ibm,dc=com"
+		bindPassword="testuserpwd"
```

Comment:
Weird this wouldn't use properties for bindDN and bindPassword for ldap.server.2.